app/pages/search/controls/SelectControl.js

```diff
@@ -2,36 +2,78 @@ import React, { PropTypes } from 'react';
 import ControlLabel from 'react-bootstrap/lib/ControlLabel';
 import FormGroup from 'react-bootstrap/lib/FormGroup';
 import Select from 'react-select';
+import isArray from 'lodash/isArray';
 
 import { injectT } from 'i18n';
 
-function SelectControl({ id, isLoading, label, onConfirm, options, t, value }) {
-  return (
-    <div className="app-SelectControl">
-      <FormGroup controlId={id}>
-        <ControlLabel>{label}</ControlLabel>
-        {!isLoading && <Select
-          clearable
-          name={id}
-          onChange={option => onConfirm(option ? option.value : '')}
-          options={options}
-          placeholder={t('common.select')}
-          searchable
-          value={value || ''}
-        />}
-      </FormGroup>
-    </div>
-  );
-}
+class SelectControl extends React.Component {
+  getValue = (value, options) => {
+    if (isArray(value)) {
+      return value.map(item => options.find(option => option.value === item));
+    }
+
+    return options.find(option => option.value === value);
```

Comment:
What if this results in `undefined`, does `react-select` handle it as a value?

Reply:
Yes, i have the case in the test file, if the value is not part of options, it just get ignored